src/main/scala/fs2/async/package.scala

```diff
@@ -0,0 +1,39 @@
+package fs2
+
+import fs2.util.Catchable
+
+
+/**
+ * Created by pach on 10/10/15.
+ */
+package object async {
+
+
+  /**
+   * Create a new continuous signal which may be controlled asynchronously.
+   * Note that this would block any resulting processes (discrete, continuous) until any signal value is set.
+   */
+  def signalUnset[F[_]:AsyncExt,A]: Stream[F,mutable.Signal[F,A]] =
```

Comment:
As soon as you apply a pipe, though, you are kind of just converting the signal to a pair of streams. Maybe that is ok but it seems easy to misuse.

Again for coordination like that I'd just use a separate Boolean signal, and have a combinator that is sort of the opposite of interrupt ("gated"?) that starts pulling from another stream only when the boolean switches to true.